skopt/optimizer/optimizer.py

```diff
@@ -148,21 +148,27 @@ def __init__(self, dimensions, base_estimator,
                 self._cat_inds.append(ind)
             else:
                 self._non_cat_inds.append(ind)
-        self._check_arguments(base_estimator, n_random_starts, acq_optimizer)
+
+        if n_random_starts != 10:
```

Comment:
A practice is to change the class instantiation to `n_random_starts=None`, so that we raise a warning when Optimizer(n_random_starts=10, ....) also raises a warning.